policykit/policyengine/models.py

```diff
@@ -229,6 +234,22 @@ def get_number_votes(self, value=0, users=None):
         else:
             votes = NumberVote.objects.filter(number_value=value, proposal=self)
         return votes
+    
+    def get_total_vote_count(self, vote_type, vote_number = 1, users = None):
```

Comment:
I don't think this should be on the datastore object. shouldn't it be on the proposal object?

Reply:
oops yes sorry!! this was a mistake